## File Checker
This code checks to see how many files from one directory is in another directory

In [3]:
import os


In [22]:
direct1 = r'C:\Users\rb01243\OneDrive - University of Surrey\Documents\GitHub\C-YOLOv3\v3-crowd\data\datasets\iid-tv\images\train'
direct2 = r'C:\Users\rb01243\OneDrive - University of Surrey\Documents\GitHub\C-YOLOv3\v3-crowd\data\datasets\Zoon_50min_3perImg_IID\images\val'

count = 0
for file in os.listdir(direct1):
    file = file.split('.')
    for file2 in os.listdir(direct2):
        file2 = file2.split('.')
        if file[0] == file2[0]:
            count += 1
print(count, ': Files Found')

320 : Files Found


#### Zoon_50min_3perImg_Crowdsourced VAL compared to bcc-tv VAL           
20 files found (all files present)

#### Zoon_50min_3perImg_Crowdsourced TRAIN compared to bcc-tv TRAIN  
0 files found (old train data could be in the new val data)

#### Zoon_50min_3perImg_Crowdsourced VAL compared to bcc-tv TRAIN  
80 files found (originally in train but now in val)

#### Zoon_50min_3perImg_IID VAL compared to iid-tv VAL
20 files found (all files present)

#### Zoon_50min_3perImg_IID VAL compared to iid-tv TRAIN
320 files found (all files present)